src/2048-expert/2048-expert.css

```diff
@@ -0,0 +1,80 @@
+/*
+Author: Zihan Zhong
+ */
+
+ * {
+    margin: 0;
+    padding: 0;
+ }
+
+ html {
+    user-select:none;
+ }
+
+
+ /* colour scheme follow the convention of 2048 games */
+ body {
+   height:100%;
```

Comment:
Please keep the spacing consistent. I think the convention this repository has adopted is a single space after each semi-colon i.e. `height: 100%;`.